README.md

```diff
@@ -160,5 +162,5 @@ Pull requests and stars are always welcome. For bugs and feature requests, [plea
 
 ## License
 
-Copyright © 2020 [Maykin Media](https://www.maykinmedia.nl/)
+Copyright © 2021 [Maykin Media](https://www.maykinmedia.nl/)
```

Comment:
Don't copyrights work with the first year that it was created, so you can leave this on 2020?

Reply:
Mm, seems you're right. I should revert this back to 2019 then.